style.css

```diff
@@ -215,3 +215,35 @@ p {
 .post-categories a {
    display: none;
 }
+
+.page .post-5 {
```

Comment:
You're styling posts/pages per post ID. This is not sustainable and you shouldn't be having to add the exact same styles to each specific post. This can be generalized.